### ISSUE TITLE
Assert failed when opening a file

### ISSUE THREAD

**Issue by Mandarancio** [fabiocolacio/Marker#189]
CRITICAL warning when opening a file.

```
(marker:11808): GLib-GIO-CRITICAL **: 03:27:10.203: g_file_get_uri: assertion 'G_IS_FILE (file)' failed
```

**Comment by fabiocolacio** [comment]
Does this crash the program, or is it just a warning? Does the file get opened anyways? I have not been able to reproduce this on my system as I mentioned in #187.

Some testing with ``gdb`` might help uncover the cause of this.

**Comment by Mandarancio** [comment]
It is just a warning and it seems to happen all the time on my arch setup. I will try to debug it this week.

**Comment by GorrillaRibs** [comment]
I get the same behaviour (also on my arch setup), below is the output when run from a terminal and trying to open a file. When marker is opened directly with a file (either through marker $file of a file manager) all is well. Would building with debug symbols and running with gdb be useful as well?

```
(marker:21110): GLib-GIO-CRITICAL **: 15:51:06.347: g_file_get_uri: assertion 'G_IS_FILE (file)' failed

(marker:21110): Gtk-CRITICAL **: 15:51:06.700: gtk_stack_add_named: assertion 'GTK_IS_STACK (stack)' failed

(marker:21110): Gtk-CRITICAL **: 15:51:06.703: gtk_stack_set_visible_child_full: assertion 'GTK_IS_STACK (stack)' failed

(marker:21110): Gtk-CRITICAL **: 15:51:06.703: gtk_header_bar_set_title: assertion 'GTK_IS_HEADER_BAR (bar)' failed

(marker:21110): Gtk-CRITICAL **: 15:51:06.703: gtk_header_bar_set_subtitle: assertion 'GTK_IS_HEADER_BAR (bar)' failed

(marker:21110): Gtk-CRITICAL **: 15:51:06.703: gtk_stack_set_visible_child_full: assertion 'GTK_IS_STACK (stack)' failed

(marker:21110): Gtk-CRITICAL **: 15:51:06.703: gtk_header_bar_set_title: assertion 'GTK_IS_HEADER_BAR (bar)' failed

(marker:21110): Gtk-CRITICAL **: 15:51:06.703: gtk_header_bar_set_subtitle: assertion 'GTK_IS_HEADER_BAR (bar)' failed

(marker:21110): Gtk-CRITICAL **: 15:51:06.703: gtk_button_set_label: assertion 'GTK_IS_BUTTON (button)' failed

(marker:21110): Gtk-CRITICAL **: 15:51:06.703: gtk_button_set_label: assertion 'GTK_IS_BUTTON (button)' failed


```

**Comment by deekosta** [comment]
I encounter this issue on Arch. Fresh install of Arch, Marker is on latest available version. I get the same errors as above: 

`(marker:4473): GLib-GIO-CRITICAL **: 10:13:35.364: g_file_get_uri: assertion 'G_IS_FILE (file)' failed

(marker:4473): Gtk-CRITICAL **: 10:13:35.734: gtk_stack_add_named: assertion 'GTK_IS_STACK (stack)' failed

(marker:4473): Gtk-CRITICAL **: 10:13:35.755: gtk_stack_set_visible_child_full: assertion 'GTK_IS_STACK (stack)' failed

(marker:4473): Gtk-CRITICAL **: 10:13:35.755: gtk_header_bar_set_title: assertion 'GTK_IS_HEADER_BAR (bar)' failed

(marker:4473): Gtk-CRITICAL **: 10:13:35.755: gtk_header_bar_set_subtitle: assertion 'GTK_IS_HEADER_BAR (bar)' failed

(marker:4473): Gtk-CRITICAL **: 10:13:35.755: gtk_stack_set_visible_child_full: assertion 'GTK_IS_STACK (stack)' failed

(marker:4473): Gtk-CRITICAL **: 10:13:35.755: gtk_header_bar_set_title: assertion 'GTK_IS_HEADER_BAR (bar)' failed

(marker:4473): Gtk-CRITICAL **: 10:13:35.755: gtk_header_bar_set_subtitle: assertion 'GTK_IS_HEADER_BAR (bar)' failed

(marker:4473): Gtk-CRITICAL **: 10:13:35.755: gtk_button_set_label: assertion 'GTK_IS_BUTTON (button)' failed

(marker:4473): Gtk-CRITICAL **: 10:13:35.755: gtk_button_set_label: assertion 'GTK_IS_BUTTON (button)' failed
`

This *does* cause a crash. All works when opening a file via Nautilus.

**Comment by Mandarancio** [comment]
Issue is fixed in commit 519290a :)
